git/homework.py

```diff
@@ -78,7 +82,11 @@ def multiple_ints_with_conversion(first_value: Any, second_value: Any) -> int:
             print("Not valid input data")
         >>> "Not valid input data"
     """
-    pass
+    try:
+        return int(first_value) * int(second_value)
+    except TypeError:
+        raise ValueError
+        print('Not valid input data')
```

Comment:
This line will never work

Reply:
Just curious.
Why does test pass?

Reply:
When an error arises, the code stops working. Until this point, execution will never reach.

Reply:
But if you want ValueError to output the "Not valid input data" error, just add this to the arguments.
For example: `ValueError('Not valid input data')`

Reply:
done